personalwebsite/static/base/dist/css/base_old.css

```diff
@@ -0,0 +1,122 @@
+:root {
+    --font-default-color: #333333;
+    --font-scale: 0.95;
+
+    --gray0: #f8f8f8;
+    --gray1: #dbe1e8;
+    --gray2: #b2becd;
+    --gray3: #6c7983;
+    --gray4: #454e56;
+    --gray5: #2a2e35;
+    --gray6: #12181b;
+}
+
+@media (min-width: 768px) {
+    :root {
+        --font-scale: 1;
+    }
+}
+
+@media(min-width: 992px) {
+    :root {
+        --font-scale: 1.05;
+    }
+}
+
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Cabin', sans-serif;
+    /* color: var(--font-default-color); */
```

Comment:
Is this needed anymore?

Reply:
No, I actually forgot to remove the file since is no longer being used.